lang/cs/Org.Apache.REEF.Network/Group/Task/Impl/CommunicationGroupClient.cs

```diff
@@ -148,7 +158,9 @@ public IScatterSender<T> GetScatterSender<T>(string operatorName)
         /// <returns>The ScatterReceiver</returns>
         public IScatterReceiver<T> GetScatterReceiver<T>(string operatorName)
         {
-            return GetOperatorInstance<ScatterReceiver<T>>(operatorName);
+            var o = GetOperatorInstance<ScatterReceiver<T>>(operatorName);
```

Comment:
Couldn't the `WaitForRegistration()` call be done within `GetOperatorInstance`? That way, the change would be smaller.